package.json

```diff
@@ -43,7 +43,8 @@
   "dependencies": {
     "carbon-components-react": "^7.18.0",
     "classnames": "^2.2.6",
-    "react": "^16.8.6"
+    "react": "^16.8.6",
+    "react-copy-to-clipboard": "^5.0.1"
```

Comment:
fyi, this change was bc of an eslint error saying it should be a dependency not a devDependency